### PR TITLE
fix(ontology): Sub-properties of link values aren't created correctly (DEV-426)

### PR DESCRIPTION
**Pull request by irinaschubert**
resolves DEV-426

### CODE REVIEW

**Review by BalduinLandolt — APPROVED** [review]

File: test_data/ontologies/freetest-onto.ttl [hunk 1]
```diff
@@ -1,181 +1,270 @@
...
+                                                                                                owl:minCardinality
+                                                                                                        "0"^^xsd:nonNegativeInteger ;
+                                                                                                salsah-gui:guiOrder
+                                                                                                        "5"^^xsd:nonNegativeInteger ] ;
+    knora-base:resourceIcon                                                                   "thing.png" ;
+    rdfs:label                                                                                "FT de"@de, "FT fr"@fr,
+                                                                                              "FT it"@it, "FT en"@en ;
+    rdfs:comment                                                                              """A comment for FT."""@de .
+
+
```
Comment (mpro7) [inline test_data/ontologies/freetest-onto.ttl:121]: Can this part be improved regarding formatting?

File: test_data/ontologies/freetest-onto.ttl [hunk 2]
```diff
@@ -1,181 +1,270 @@
...
+                                                                                          "1"^^xsd:nonNegativeInteger ],
+                                                                         [ rdf:type       owl:Restriction ;
+                                                                           owl:onProperty :hasScientificAuthorValue ;
+                                                                           owl:minCardinality
+                                                                                          "0"^^xsd:nonNegativeInteger ;
+                                                                           salsah-gui:guiOrder
+                                                                                          "2"^^xsd:nonNegativeInteger ] ;
+    rdfs:label                                                           "Wissenschaftliches Buch"@de,
+                                                                         "Scientific book"@en ;
+    rdfs:comment                                                         """A comment for a scientific book"""@en .
```
Comment (mpro7) [inline test_data/ontologies/freetest-onto.ttl:232]: bad formatting

File: test_data/ontologies/freetest-onto.ttl [hunk 3]
```diff
@@ -1,181 +1,270 @@
...
+                                                                                          "3"^^xsd:nonNegativeInteger ],
+                                                                         [ rdf:type       owl:Restriction ;
+                                                                           owl:onProperty :hasOtherCULPValue ;
+                                                                           owl:minCardinality
+                                                                                          "0"^^xsd:nonNegativeInteger ;
+                                                                           salsah-gui:guiOrder
+                                                                                          "4"^^xsd:nonNegativeInteger ] ;
+    rdfs:label                                                           "CULP de"@de, "CULP fr"@fr, "CULP it"@it,
+                                                                         "CULP en"@en ;
+    rdfs:comment                                                         """A comment for CULP."""@de .
```
Comment (mpro7) [inline test_data/ontologies/freetest-onto.ttl:270]: bad formatting

**Review by mpro7 — APPROVED** [review]
Looks ok, but check if you can somehow improve TTL formatting.

**Review by subotic — APPROVED** [review]

File: test_data/ontologies/freetest-onto.ttl (hunk 1 above)
Comment (irinaschubert) [reply test_data/ontologies/freetest-onto.ttl:121]: As discussed, we are going to solve the formatting in another PR/task.

**Pull request merged**